azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1049,6 +1068,10 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 
 	values["network_plugin"] = profile.NetworkPlugin
 
+	if profile.NetworkPolicy != "" {
+		values["network_policy"] = profile.NetworkPolicy
```

Comment:
I think we should be casting this to a string?
```suggestion
		values["network_policy"] = string(profile.NetworkPolicy)
```